src/app/lib/config/toggles/index.js

```diff
@@ -72,6 +75,9 @@ const toggles = {
       value:
         '(afaanoromoo|afrique|amharic|arabic|archive|azeri|bengali|burmese|cymrufyw|gahuza|gujarati|hausa|hindi|igbo|indonesia|japanese|korean|kyrgyz|marathi|mundo|naidheachdan|nepali|news|pashto|persian|pidgin|portuguese|punjabi|russian|scotland|serbian|sinhala|somali|swahili|tamil|telugu|thai|tigrinya|turkce|ukchina|ukrainian|urdu|uzbek|vietnamese|yoruba|zhongwen)',
     },
+    socialEmbed: {
+      enabled: false,
```

Comment:
Is this going to be enabled on test when ares release their api with social embed support?

Reply:
Yes, I suppose it won't hurt to toggle it on test now. 👍